### PR TITLE
refactor(bin/webpack): use `err.name` for error handling

### PR DESCRIPTION
**Pull request by michael-ciniawsky**
**What kind of change does this PR introduce?**

- [x] Refactor

**Did you add tests for your changes?**

- [x] Yes

**Summary**

One less `require()` call

**Does this PR introduce a breaking change?**

- [x] No

### CODE REVIEW

**Comment by michael-ciniawsky** [comment]
The failure is to be expected atm, due to [schema-utils (#14)](https://github.com/webpack-contrib/schema-utils/pull/14)

**Comment by michael-ciniawsky** [comment]
I found a better solution (to handle it directly in `schema-utils` instead).

This PR contains only a small refactor  to error handling now. I will leave it open, but feel free to close :)

File: bin/webpack.js [hunk 1]
```diff
@@ -333,16 +333,19 @@ yargs.parse(process.argv.slice(2), (err, argv, output) => {
 		var compiler;
 		try {
 			compiler = webpack(options);
-		} catch(e) {
-			var WebpackOptionsValidationError = require("../lib/WebpackOptionsValidationError");
-			if(e instanceof WebpackOptionsValidationError) {
+		} catch(err) {
+			if(err.name === "WebpackOptionsValidationError") {
```
Comment (sokra) [inline bin/webpack.js]: Could you write a test for this? A simple one in `test/binCases`.

Comment (michael-ciniawsky) [reply bin/webpack.js]: Yup, done

**Comment by sokra** [comment]
Thanks

**Pull request merged**